docs/Azure_IoT_Hub_integration/Device_operations/Upgrading_firmware.md

```diff
@@ -0,0 +1,117 @@
+# Upgrading device firmware
+
+If you would like to upgrade the firmware of devices using the Azure IoT Hub, follow the instruction below.
```

Comment:
I think that we should add here 2 very important statements:
1. How this fota process works - We schedule the fota task from azure, the Coiote downloads the firmware and adds it to resources in Coiote, then schedule XML task - do the update. 
2. How this firmware is recognized in Coiote - we use global identifiers (it will be changed in the future) so you should use format domainName + some random value. When you use the same name, Coiote takes the old resource.

Reply:
added as suggested